.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -0,0 +1,59 @@
+name: Cloud TPU nightly
```

Comment:
[Here for threading]

You will find that getting notifications is *way* harder than it should be. This is our issue for it: https://github.com/iree-org/iree/issues/9857. Would love to combine efforts on this though. No point reinventing solutions here.

Reply:
Oooh filing issues! That's actually a great idea. We should do that. I'm less psyched about chat integration. I think it's bad the amount of stuff that has moved to chat services...